### PR TITLE
feat(helm-chart) create helm chart version 0.1.0

### LINKED ISSUE #4: Create Helm chart

### PR DESCRIPTION
**Pull request by wesjdj**
Create Helm chart version 0.1.0 and GitHub Action to push & build Docker image

closes #4

### CODE REVIEW

**Comment by wesjdj** [comment]
when this has been merged I will publish helm chart version 0.1.0 with chartpress

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: docker
```
Comment (aledegano) [inline .github/workflows/docker.yml:1]: perhaps we could use a more telling name, like "build container image"?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        uses: actions/checkout@v3
+      -
+        name: Fetch all tags
+        run: git fetch --force --tags
+      -
+        name: get version tag
+        run: echo "TAG_VERSION=$(echo ${GITHUB_REF#refs/*/})" >> $GITHUB_ENV
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```
Comment (aledegano) [inline .github/workflows/docker.yml:23]: why do we need to setup QEMU? we are only compiling for x86_64, are we?

File: helm-chart/taweret/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: taweret
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```
Comment (aledegano) [inline helm-chart/taweret/Chart.yaml:5]: I think we can remove all the comments here, they look like a guide for the first time this file is setup.

File: .github/workflows/docker.yml (hunk 1 above)
Comment (wesjdj) [reply .github/workflows/docker.yml]: done!

File: .github/workflows/docker.yml (hunk 2 above)
Comment (wesjdj) [reply .github/workflows/docker.yml]: true, I've removed it now

File: helm-chart/taweret/Chart.yaml (hunk 3 above)
Comment (wesjdj) [reply helm-chart/taweret/Chart.yaml]: yep, removed

File: .github/workflows/docker.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: Build & push container image
```
Comment (aledegano) [inline .github/workflows/docker.yml:1]: Can you change also the filename please? I am pre-approving, ready to go!

**Review by aledegano — DISMISSED** [review]

File: .github/workflows/docker.yml (hunk 4 above)
Comment (wesjdj) [reply .github/workflows/docker.yml]: done!

**Review by aledegano — APPROVED** [review]

**Pull request merged**